src/modules/colorPicker/ColorPickerUI/Views/MainView.xaml

```diff
@@ -1,20 +1,23 @@
-﻿<UserControl x:Class="ColorPicker.Views.MainView"
-             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
-             xmlns:local="clr-namespace:ColorPicker.Views"
-             xmlns:viewModel="clr-namespace:ColorPicker.ViewModels"
-             mc:Ignorable="d" d:DesignHeight="64" 
-             d:DataContext="{d:DesignInstance viewModel:MainViewModel, IsDesignTimeCreatable=True}">
+﻿<UserControl
+    x:Class="ColorPicker.Views.MainView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="clr-namespace:ColorPicker.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:viewModel="clr-namespace:ColorPicker.ViewModels"
+    d:DataContext="{d:DesignInstance viewModel:MainViewModel,
+                                     IsDesignTimeCreatable=True}"
+    d:DesignHeight="64"
+    mc:Ignorable="d">
     <Grid Background="Transparent">
 
-        <Border BorderBrush="{DynamicResource WindowBorderBrush}"
-                Margin="12"
-                BorderThickness="1"
-                Background="{DynamicResource PrimaryBackgroundBrush}"
-                CornerRadius="4">
-            <!-- Disabled due to the poor performance of drawing shadows in WPF -->
+        <Border
+            Background="{DynamicResource ApplicationBackgroundBrush}"
+            BorderBrush="{DynamicResource SurfaceStrokeColorDefaultBrush}"
+            BorderThickness="1"
+            CornerRadius="8">
+            <!--  Disabled due to the poor performance of drawing shadows in WPF  -->
```

Comment:
Is this comment still valid?

Reply:
There's still commented code that can stay as we might want to reconsider this as some point